joomla/joomla-cms
# [4.2] Update deleted files and folders for 4.2-beta1

Pull Request for Issue # .

### Summary of Changes

Update the lists of files and folders to be deleted on update by script.php to the latest changes in the 4.2-dev branch in preparation of the upcoming 4.2.0 Beta 1.

Some folders have already been added with PR #37912 , but it needs to delete also all stuff inside these folders, and the folders have to be in alphabetically descending order in the list so subfolders are deleted before their parent folders.

#### Changes from PR #36751 - see https://github.com/joomla/joomla-cms/pull/36751/files#diff-6e79b8dcfcf48c49f12b78d5d2c631b91c199dbfc39181c7f80e2d08ca703548 :
- '/media/com_finder/js/index-es5.js'
- '/media/com_finder/js/index-es5.min.js'
- '/media/com_finder/js/index-es5.min.js.gz'
- '/media/com_finder/js/index.js'
- '/media/com_finder/js/index.min.js'
- '/media/com_finder/js/index.min.js.gz'

#### Changes from PR #37788 - see https://github.com/joomla/joomla-cms/pull/37788/files#diff-ae190cb5a3211dd66bd21b04b0cf008d7667f71095e580a92467352a5bfd6cb8 :
- '/plugins/actionlog/joomla/joomla.php'

#### Changes from PR #37912 - see https://github.com/joomla/joomla-cms/pull/37912/files#diff-360f9716fcf3fdb5a8d9a109c8c16460cc8b47e25ad08ed4b4ba9d3916f7e9f8 :
- '/media/com_users/js/two-factor-switcher-es5.js'
- '/media/com_users/js/two-factor-switcher-es5.min.js'
- '/media/com_users/js/two-factor-switcher-es5.min.js.gz'
- '/media/com_users/js/two-factor-switcher.js'
- '/media/com_users/js/two-factor-switcher.min.js'
- '/media/com_users/js/two-factor-switcher.min.js.gz'

#### Also from PR #37912 :
- '/plugins/twofactorauth/totp/postinstall/actions.php'
- '/plugins/twofactorauth/totp/tmpl/form.php'
- '/plugins/twofactorauth/totp/totp.php'
- '/plugins/twofactorauth/totp/totp.xml'
- '/plugins/twofactorauth/yubikey/tmpl/form.php'
- '/plugins/twofactorauth/yubikey/yubikey.php'
- '/plugins/twofactorauth/yubikey/yubikey.xml'

### Testing Instructions

Code review.

Or if you want to make a real test, update a 4.2.0-alpha3 to the last 4.2 nightly build to get the actual result, and update a 4.2.0-alpha3 to the update package built by Drone for this PR to get the expected result.

### Actual result BEFORE applying this Pull Request

Not all files and folders mentioned in the description above are deleted when updating from 4.2.0-alpha3.

### Expected result AFTER applying this Pull Request

All files and folders mentioned in the description above all deleted when updating from 4.2.0-alpha3.

### Documentation Changes Required

None.
